packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FLTClusterManagersController.m

```diff
@@ -0,0 +1,144 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "FLTClusterManagersController.h"
+#import "FLTGoogleMapJSONConversions.h"
+#import "GoogleMarkerUtilities.h"
+
+@interface FLTClusterManagersController ()
+
+@property(strong, nonatomic)
+    NSMutableDictionary<NSString *, GMUClusterManager *> *clusterManagerIdToManagers;
```

Comment:
`ID` rather than `Id`

Reply:
Renamed this to `clusterManagerIdentifierToManagers`